migrations/1677284495299_locations.ts

```diff
@@ -6,15 +6,18 @@ export const shorthands: ColumnDefinitions | undefined = undefined;
 export function up(pgm: MigrationBuilder): void {
   pgm.createTable('locations', {
     id: {
-      type: 'serial',
+      type: 'bigserial',
       primaryKey: true,
     },
     inscription_id: {
-      type: 'int',
+      type: 'bigint',
+    },
+    genesis_id: {
+      type: 'text',
       notNull: true,
     },
     block_height: {
-      type: 'int',
+      type: 'bigint',
```

Comment:
this could stay a `int`, no?

Reply:
We're standardizing this on all of the API int columns, but no worries I always make the appropriate conversion to int when displaying in endpoint responses